src/test-derivatives.cpp

```diff
@@ -80,7 +81,8 @@ context("derivatives_sp_exp struct works as expected") {
       0, 0.5, 1,
       0.5, 0, 0.5,
       1, 0.5, 0;
-    auto sigma = sp.get_sigma(dist);
+    std::vector<int> v(3);
```

Comment:
should we better explicitly say which values should be in `v`?

Reply:
for spatial derivatives, v is not used at all; so this is can be any uninitialized value; or we can make it 0 length vector